config/database.yml

```diff
@@ -20,6 +20,14 @@ test:
   variables:
     idle_in_transaction_session_timeout: 0
 
+deployed_development:
+  <<: *default
+  sslmode: require
+
+staging:
+  <<: *default
+  sslmode: require
+
```

Comment:
Do we need different configs for `deployed_development` and `staging`?

Reply:
Different API keys, so different environments, so different configs